mongodb/dev.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mongodb-dev
+spec:
+  containers:
+  - name: mongodb-database
+    image: mongo:4.4.18
+  - name: mongodb-backup
+    image: sebastienbonami/mongodb-backup:latest
```

Comment:
whenever possible, I would avoid putting a user account here.

Other users wishing to test it would not have access to it anyways making it of limited use.

In Linux with a local kubernetes, it is easy enough for a user to build an image locally anyways.